SEED-platform/seed
# 3052 refactor/top right nav update

#### Any background context you want to provide?
See issue #3050

#### What's this PR do?
- This PR updates the function and appearance of the `_header` navigation buttons. 
- The `+` button will now direct users to the organization settings of the current organization based on their authorization role 
- The current organization has been reformatted to show the current (selected) organization
- When the user hovers over an organization in the drop-down the organization id will appear above the mouse


#### How should this be manually tested?
- From any page, use the `+` button to navigate to the organization's various settings. Ensure it is the correct organization (check URL for organization id) 
- with an Admin account (role_level = 20) see all child organizations
- with a nonauthorized user check the available settings and organizations. If they do not have permission (role_level = 0) the `+` button will disappear. Partially authorized users (role_level = 10) will be able to click the `+` but will only have access to 'members' and 'more...'
- Hovering over the organization dropdown will show the organization id for all users

#### What are the relevant tickets?
#3052 

### Screenshots (if appropriate)
#### Admin  (`role_level` = 20) hover over organization dropdown 
![Screen Shot 2022-01-05 at 10 23 55 AM](https://user-images.githubusercontent.com/58446472/148264091-ca7302b8-c13c-4992-910a-a84944c92c7e.png)

![Screen Shot 2022-01-05 at 10 23 45 AM](https://user-images.githubusercontent.com/58446472/148264092-6ab774c2-60dd-411b-98a7-3db7c00cd6ce.png)

#### Admin (`role_level` = 20) `+` functionality
![Screen Shot 2022-01-05 at 10 23 24 AM](https://user-images.githubusercontent.com/58446472/148264093-7392bfa8-40d3-4f27-b3f5-bb9d73a43379.png)

#### User (`role_level` = 10) `+` functionality
![Screen Shot 2022-01-05 at 10 24 57 AM](https://user-images.githubusercontent.com/58446472/148264087-c40ae2a4-437d-48f0-88db-0d484fd70ec4.png)

#### User (`role_level` = 10) hover over organization dropdown
![Screen Shot 2022-01-05 at 10 47 45 AM](https://user-images.githubusercontent.com/58446472/148264670-e36223bd-cbc8-41d1-8179-762c3acb1390.png)

#### Non-authorized user (`role_level` = 0) hover over organization dropdown. 
Note: `+` hidden from non-authorized user
![Screen Shot 2022-01-05 at 10 48 15 AM](https://user-images.githubusercontent.com/58446472/148264668-e73bde49-5553-41c9-9665-8fb9c7052171.png)
